src/rsz/test/repair_design3_verbose.ok

```diff
@@ -14,16 +14,15 @@
 Iteration | Resized | Buffers | Nets repaired | Remaining
 ---------------------------------------------------------
         0 |       0 |       0 |             0 |      1002
-      100 |      98 |       0 |             0 |       902
-      200 |     198 |       0 |             0 |       802
-      300 |     298 |       0 |             0 |       702
-      400 |     398 |       0 |             0 |       602
-      500 |     498 |       0 |             0 |       502
-      600 |     598 |       0 |             0 |       402
-      700 |     698 |       0 |             0 |       302
-      800 |     798 |       0 |             0 |       202
-      900 |     898 |       0 |             0 |       102
-     1000 |     998 |       0 |             0 |         2
-    final |    1000 |       0 |             0 |         0
+      100 |       0 |       0 |             0 |       902
+      200 |       0 |       0 |             0 |       802
+      300 |       0 |       0 |             0 |       702
+      400 |       0 |       0 |             0 |       602
+      500 |       0 |       0 |             0 |       502
+      600 |       0 |       0 |             0 |       402
+      700 |       0 |       0 |             0 |       302
+      800 |       0 |       0 |             0 |       202
+      900 |       0 |       0 |             0 |       102
+     1000 |       0 |       0 |             0 |         2
+    final |       0 |       0 |             0 |         0
```

Comment:
Is it right that there are zero changes here?

Reply:
If I understand it correctly, this design has no violations to repair, so the resizing is never executed. That's why we have zeros now.